yellow-paper/sidebars.js

```diff
@@ -21,35 +21,55 @@ const sidebars = {
   yellowPaperSidebar: [
     "intro",
     {
-      "Addresses & keys": [
-        "addresses-and-keys/index",
+      label: "Addresses & Keys",
+      type: "category",
+      link: { type: "doc", id: "addresses-and-keys/index" },
+      items: [
         "addresses-and-keys/specification",
         "addresses-and-keys/precompiles",
         "addresses-and-keys/diversified-and-stealth",
       ],
-      State: [
-        "state/index",
+    },
+    {
+      label: "State",
+      type: "category",
+      link: { type: "doc", id: "state/index" },
+      items: [
         "state/tree-implementations",
         "state/archive",
         "state/note-hash-tree",
         "state/nullifier-tree",
         "state/public-data-tree",
       ],
-      Transactions: [
-        "transactions/index",
+    },
+    {
+      label: "Transactions",
+      type: "category",
+      link: { type: "doc", id: "transactions/index" },
+      items: [
         "transactions/local-execution",
         "transactions/public-execution",
         "transactions/tx-object",
         "transactions/validity",
       ],
-      Bytecode: ["bytecode/index"],
-      "Contract Deployment": [
-        "contract-deployment/index",
-        "contract-deployment/classes",
-        "contract-deployment/instances",
-      ],
-      Calls: [
-        "calls/index",
+    },
+    {
+      label: "Bytecode",
```

Comment:
Why did you remove the classes and instances from here? Oh, seems like they were just moved into the "contract" deployment. Why? 
Should the bytecode and contract deployment be connected or separate pages entirely?

Reply:
> Why did you remove the classes and instances from here? Oh, seems like they were just moved into the "contract" deployment. Why?

I don't think this PR does move them around, vs what's currently in `master`?
 
'bytecode' is separate from 'contract deployment', just because bytecode is of interest in other contexts, e.g. execution (not just at the time of deployment).